src/main/scala/com/databricks/spark/sql/perf/tpcds/SimpleQueries.scala

```diff
@@ -138,7 +138,38 @@ trait SimpleQueries extends Benchmark {
               |  i_item_id
               |limit 100
               |-- end query 1 in stream 0 using template query7.tpl
-            """.stripMargin)
+            """.stripMargin),
+            
+      ("q7-bigtable-selfjoin-1",   """
+                                   |-- 1.5 is slower than 1.4.1 
```

Comment:
Instead of mentioning that 1.5 is slower than 1.4.1, it will be helpful to mention that this query will generate much more rows than input rows because the join condition will yield many matches.